native/cocos/network/HttpClient-apple.mm

```diff
@@ -345,6 +346,7 @@ static int processTask(HttpClient *client, HttpRequest *request, NSString *reque
 
 HttpClient::~HttpClient() {
     CC_SAFE_RELEASE(_requestSentinel);
+    CC_SAFE_DELETE(_threadPool);
```

Comment:
Deleting thread pool will join all threads. If network connection is bad, join may cost a lot of time. I think keeping it as a global variable is ok.

Reply:
make sense. Done!

Reply:
Should refactor it after network module is added. Threads are a limitation resource.